themes/default/content/case-studies/atlassian.md

```diff
@@ -0,0 +1,107 @@
+
+Pulumi also simplified cloud infrastructure management and provided clear visibility into all active resources. Mike said, “Before Pulumi, we had a manager for state files that was a complete pain. We ran into weird issues and errors, and it was difficult to ask questions, such as how many developer instances were running. Now, Pulumi manages all the state files for us and gives us that visibility.
+
+Mike also gave an example of how easy it was to deploy changes across all instances with confidence. He says, “Subnets have a maximum number of IP addresses associated with them. We ran into an issue where there would be failures because addresses were selected randomly. I corrected the issue and could confidently deploy it by just writing a test to make sure that select logic worked. And because we're using Python, I was able to just use the AWS Python API to figure out which subnets were full! That's simply not possible using a DSL.”
+
+### Result
+
+Using Pulumi, with its support for Python, Mike quickly and easily built a repeatable pipeline that deploys development environments for the Sydney team despite having limited knowledge of AWS. He also now understood how useful Pulumi could be in improving the developer experience at Bitbucket and how easy it was to use. He began to look for other ways to use Pulumi.
+
+## Second Challenge
+
+### Improving Developer Productivity with Self-Service Cloud Infrastructure
+
+Shortly afterward, Mike reviewed a survey sent to the Bitbucket developers, asking them how they spent their time. He was surprised to learn that developers were spending 20% of their workweek maintaining their private Bitbucket development environments.
+
+Again, part of the problem rested with the old tool, whose DSL was unfamiliar to most developers. Because it lacked standard programming capabilities, it made the code verbose and difficult to understand. Mike knew that he could streamline the entire workflow with Pulumi and make infrastructure more accessible and simpler to use for developers who understand Python.
+
+To make things simple and turnkey for developers, the DevSpeed team built a self-service dashboard that created and configured new instances. Developers enter a few pieces of information, such as what they want to call the instance, what region to use, and an SSH key. Then, a CI/CD pipeline using Bitbucket Pipelines and Bamboo calls Pulumi to deploy a complete developer environment.
+
+Once the instance is ready, the developer simply SSH’s into it, and it will look exactly like Bitbucket. The developer can make the changes they want and then create a pull request to merge their work into a main branch. Developers can also share the URL of the instance with teammates to see the changes when they review the code.
+
+Here is what an instance, spun up with Pulumi, looks like:
+
+<img class="block mx-auto md:max-w-4xl my-8" src="/images/case-studies/atlassian-diagram.png" alt="Atlassian Instance">
```

Comment:
This doesn't seem to be necessary, as the default margins give consistent spacing.

```suggestion
![Atlassian Instance](/images/case-studies/atlassian-diagram.png)
```